### PR TITLE
processing text ellipsis animation

### PR DESCRIPTION
**Pull request by aaronshiel**

### CODE REVIEW

File: client/src/hooks/graphql/use-with-upload-status.tsx [hunk 1]
```diff
@@ -50,6 +50,8 @@ export function useWithUploadStatus(
   const [uploads, setUploads] = useState<UploadTask[]>([]);
   const [isUploading, setIsUploading] = useState<boolean>(false);
   const [isPolling, setIsPolling] = useState<boolean>(false);
+  const [uploadProcessingText, setUploadProcessingText] =
```
Comment (beatthat) [inline client/src/hooks/graphql/use-with-upload-status.tsx:53]: for how we're doing this, maybe cleaner if we just provided a poll count here?

```typescript
const [pollStatusCount, setPollStatusCount]
```

...that way all the stuff about ellipses can be a concert for the presentation component, which seems a cleaner division of responsibility

**Review by beatthat — COMMENTED** [review]
Looks good. See initial question/comment inline

File: client/src/hooks/graphql/use-with-upload-status.tsx (hunk 1 above)
Comment (aaronshiel) [reply client/src/hooks/graphql/use-with-upload-status.tsx]: 3632c1d

File: client/src/hooks/graphql/use-with-upload-status.tsx [hunk 2]
```diff
@@ -81,11 +82,11 @@ export function useWithUploadStatus(
     setIsPolling(uploads.some((u) => isTaskPolling(u)));
   }, [uploads]);
 
-  //we start polling because we added an upload to our state when pressed
   useInterval(
     (isCancelled) => {
       fetchUploadTasks(accessToken)
         .then((data) => {
+          setPollStatusCount(pollStatusCount + 1);
```
Comment (beatthat) [inline client/src/hooks/graphql/use-with-upload-status.tsx:89]: is `isCancelled` I think no reason to make any additional state updates?

**Review by beatthat — CHANGES_REQUESTED** [review]
looks great. Very small q and possible change request about whether to update anything in the case of `isCancelled` after that, ready to merge

**Review by beatthat — APPROVED** [review]

**Pull request merged**